wasmer/memory.go

```diff
@@ -69,10 +70,12 @@ func (memory *Memory) Grow(numberOfPages uint32) error {
 		return nil
 	}
 
+	runtime.LockOSThread()
+	defer runtime.LockOSThread()
```

Comment:
I think there is a typo:

```suggestion
	defer runtime.UnlockOSThread()
```

Reply:
Yes. Thanks! Fixed!